che-incubator/configbump
# chore: add release scripts so configbump can follow che releases (and stay current w/ CVE fixes)

### What does this PR do?

Add release scripts for version 7.75.x

Co-authored-by: Mykhailo Kuznietsov <mkuznets@redhat.com>
Co-authored-by: Dmytro Nochevnov <dnochevn@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/21988

Replaces: https://github.com/che-incubator/configbump/pull/76

### How to test this PR?

Merge it ! then run the new action from https://github.com/che-incubator/configbump/actions

Once merged, see TODO in https://github.com/eclipse-che/che-release/pull/80 which we can then update and merge before 7.75.0 release day

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
